src/model/Model_CurrencyHistory.cpp

```diff
@@ -85,41 +85,68 @@ int Model_CurrencyHistory::addUpdate(const int& currencyID, const wxDate& date,
 /** Return the rate for a specific currency in a specific day*/
 double Model_CurrencyHistory::getDayRate(const int& currencyID, const wxString& DateISO)
 {
-    if (currencyID == Model_Currency::GetBaseCurrency()->CURRENCYID)
+    if (currencyID == Model_Currency::GetBaseCurrency()->CURRENCYID || currencyID == -1)
         return 1;
     
     wxDateTime Date;
     Date.ParseDate(DateISO);
```

Comment:
What we do if DateISO>today?
Can user enter predicted/expected rates into Currency History?
Is it better to use today+1 value then real today-2 one?

Reply:
I'm thinking about blocking entering future values